curriculum/outlines/Linear DS.md

```diff
@@ -0,0 +1,39 @@
+# Linear DS
+---
+## Outline
+1. Static Array
+	 1. 1D Static Array
+	 2. 2D Static Array
+	 3. 3D Static Array
+2. Dynamically-Resizeable Array
```

Comment:
Highlight important things as well like (think of anything I have missed):
- important functions: get, add, set, .. others I don't remember. 
- behind the scenes: Resizing complexity

A rule of thumb. Always highlight anything that shouldn't be missed in the session. Give an explanation only if it's not provided in the linked tutorial.